Include sys/select.h as defined by POSIX to have access to fd_set

diff --git a/gdraw/gdraw.c b/gdraw/gdraw.c
index 24591b7..ab045d0 100644
--- a/gdraw/gdraw.c
+++ b/gdraw/gdraw.c
@@ -36,6 +36,7 @@
 #  include <sys/select.h>
 #endif
 
+#include <sys/select.h>
 
 /* Functions for font metrics:
     rectangle of text (left side bearing of first char, right of last char)
diff --git a/gdraw/gxdraw.c b/gdraw/gxdraw.c
index 3cbf6af..b04ed9b 100644
--- a/gdraw/gxdraw.c
+++ b/gdraw/gxdraw.c
@@ -52,6 +52,7 @@
 #include <signal.h>		/* error handler */
 #include <sys/time.h>		/* for timers & select */
 #include <sys/types.h>		/* for timers & select */
+#include <sys/select.h>
 
 enum cm_type { cmt_default=-1, cmt_current, cmt_copy, cmt_private };
 

